Data-Structures/src/main/java/linkedlist/LinkedList.java

```diff
@@ -70,5 +70,22 @@ public void insertAfter(T value, T target){
     }
   }
 
+  public String kthFromEnd(int k){
+    this.current = head;
+    int length = 0;
+    while(this.current != null) {
+      this.current = current.next;
+      length++;
+    }
+    if(length < k){
+      return "Not there";
```

Comment:
Food for thought: What if "Not there" is the value of the node `kth` from the end?